tests/test_semaphore_tracker.py

```diff
@@ -0,0 +1,209 @@
+"""Tests for the SemaphoreTracker class"""
+import errno
+import gc
+import io
+import os
+import pytest
+import re
+import signal
+import sys
+import time
+import warnings
+import weakref
+
+from loky import ProcessPoolExecutor
+import loky.backend.semaphore_tracker as semaphore_tracker
+from loky.backend.semlock import sem_unlink
+from loky.backend.context import get_context
+
+
+def get_sem_tracker_pid():
+    semaphore_tracker.ensure_running()
+    return semaphore_tracker._semaphore_tracker._pid
+
+
+class TestSemaphoreTracker:
+
+    @pytest.mark.skipif(sys.platform == "win32",
+                        reason="no semaphore_tracker on windows")
+    def test_child_retrieves_semaphore_tracker(self):
+        parent_sem_tracker_pid = get_sem_tracker_pid()
+        executor = ProcessPoolExecutor(max_workers=2)
+        child_sem_tracker_pid = executor.submit(get_sem_tracker_pid).result()
```

Comment:
To be completely clean, please shutdown the executor here.
Also, you could use only one worker here to make the test quicker.